bcipy/acquisition/devices.json

```diff
@@ -28,11 +30,12 @@
     {
         "name": "DSI",
         "content_type": "EEG",
-        "channels": [ "P3", "C3", "F3", "Fz", "F4", "C4", "P4",  "Cz", "A1", "Fp1", "Fp2", "T3",
-            "T5", "O1", "O2", "F7", "F8", "A2", "T6", "T4", "TRG" ],
+        "channels": [ "P3", "C3", "F3", "Fz", "F4", "C4", "P4", "Cz", "Pz", "Fp1", "Fp2", "T3",
+            "T5", "O1", "O2", "X3", "X2", "F7", "F8", "X1", "A2", "T6", "T4", "TRG" ],
         "connection_methods": [ "TCP", "LSL" ],
         "sample_rate": 300.0,
-        "description": "Alias to DSI-24 for backwards compatibility and demos."
+        "description": "Alias to DSI-24 for backwards compatibility and demos.",
+        "excluded_from_analysis": ["Pz", "TRG", "X1", "X2", "X3"]
```

Comment:
Should we error or warn on invalid channel names? I'm not sure we want the app to fail, but it may avoid some silent failing (does capitalization matter?). 

"excluded_from_analysis": ["Pz", "TG", "X1", "X2", "X3"]. ---->

Warning "Excluded channel TG not found in self.channels={[]}, Skiping."